cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/monitor/LoadMonitor.java

```diff
@@ -600,19 +603,38 @@ private void populateLoad(Cluster kafkaCluster,
    * @param valuesAndExtrapolations the values and extrapolations of the leader replica.
    * @param partitionInfo the partition info.
    * @param isLeader whether the value is created for leader replica or follower replica.
+   * @param needToAdjustCpuUsage whether need to adjust cpu usage metric for replica.
```

Comment:
Nit: Can we make the explanation more verbose? -- i.e. what does it mean to `adjust` cpu usage metric?